saleor/graphql/product/mutations/products.py

```diff
@@ -73,8 +74,11 @@
     get_draft_order_lines_data_for_variants,
     get_used_attribute_values_for_variant,
     get_used_variants_attribute_values,
+    update_ordered_media,
 )
 
+logger = logging.getLogger(__name__)
```

Comment:
Do we need this logger?

Reply:
If we don't want to `logger.warning(msg)` I would say no :)

Reply:
We don't have `logger.warning(msg)` here or I am blind.

Reply:
So can we remove it? :)

Reply:
We have it [here](https://github.com/saleor/saleor/pull/9225/files#diff-ff3d47ae262b3250574ecd74a7750c0b04ddf0d0d84498605810753a4ba094dfR49) actually. Do we need this log, if not I would remove it :)

Reply:
LOL, I looked at the wrong file. Removed :D